tests/helpers/test_selector.py

```diff
@@ -294,6 +294,21 @@ def test_time_selector_schema(schema, valid_selections, invalid_selections):
     _test_selector("time", schema, valid_selections, invalid_selections)
 
 
+@pytest.mark.parametrize(
+    "schema,valid_selections,invalid_selections",
+    (
+        (
+            {"entity_id": "sensor.abc"},
+            ("friendly_name", "device_class"),
+            (None,),
```

Comment:
let's add some other types that are invalid.
```suggestion
            (None, True, 1),
```